lib/eslint/eslint-helpers.js

```diff
@@ -568,41 +568,6 @@ async function findFiles({
     ];
 }
 
-
-/**
- * Checks whether a file exists at the given location
- * @param {string} resolvedPath A path from the CWD
- * @throws {Error} As thrown by `fs.statSync` or `fs.isFile`.
- * @returns {boolean} `true` if a file exists
- */
-function fileExists(resolvedPath) {
-    try {
-        return fs.statSync(resolvedPath).isFile();
-    } catch (error) {
-        if (error && (error.code === "ENOENT" || error.code === "ENOTDIR")) {
-            return false;
-        }
-        throw error;
-    }
-}
-
-/**
- * Checks whether a directory exists at the given location
- * @param {string} resolvedPath A path from the CWD
- * @throws {Error} As thrown by `fs.statSync` or `fs.isDirectory`.
- * @returns {boolean} `true` if a directory exists
- */
-function directoryExists(resolvedPath) {
-    try {
-        return fs.statSync(resolvedPath).isDirectory();
-    } catch (error) {
-        if (error && (error.code === "ENOENT" || error.code === "ENOTDIR")) {
-            return false;
-        }
-        throw error;
-    }
-}
```

Comment:
I see the `directoryExists` is also defined in `cli-engine.js`, should we remove it from there instead of removing it from helpers?

https://github.com/eslint/eslint/blob/337f7ed96131d873be7ae6b010739476d0ad15e9/lib/cli-engine/cli-engine.js#L556-L565

Reply:
`eslint-helpers.js` module is intended for use by the `FlatESLint` class only.

*  `flat-eslint.js` and `eslint-helpers.js` are modules of the new config system.
* `eslint.js` and `cli-engine.js` are modules of the current eslitrc config system.

At some point, we'll remove `eslint.js` and `cli-engine.js`, and rename `flat-eslint.js` to `eslint.js` (and rename `FlatESLint` class to `ESLint` class).